benchmark-utils_test.go

```diff
@@ -224,21 +201,62 @@ func runGetObjectBenchmark(b *testing.B, obj ObjectLayer, objSize int) {
 
 }
 
-// randomly picks a character and returns its equivalent byte array.
-func getRandomByte() []byte {
```

Comment:
You can cleanup this function itself to get a random character. Since I've used this function in other places it'll break them.